genie-web/src/main/resources/db/migration/h2/V3_3_0__Merge_Jobs_Tables.sql

```diff
@@ -0,0 +1,501 @@
+/*
```

Comment:
Is this a 1:1 copy of one of the existing ones?

Reply:
I took the mysql 3.3.0 one and modified it for syntax that wasn't supported in H2 primarily the indices need to be defined separately from table definition like how postgres does it. Also constraint definitions were slightly different syntax but that's pretty much it.